cogs/roll.py

```diff
@@ -0,0 +1,40 @@
+"""Command to imitate an ingame dice roll. Can roll randomly, or return the summary of the roll."""
+from hagadias.helpers import DiceBag
```

Comment:
There is a provision in PEP8 for sorting imports, where imports of third party libraries go in the second import section: https://www.python.org/dev/peps/pep-0008/#imports

Since hagadias is now a cool third party library like the cool kids have we can put it there now :)

Reply:
Thanks for letting me know! hagadias can finally hang with the cool third parties